api/info/service.md

```diff
@@ -695,7 +695,8 @@ curl -X POST --data '{
     "cortinaXChainStopVertexID": "11111111111111111111111111111111LpoYY",
     "durangoTime": "2020-12-05T05:00:00Z",
     "etnaTime": "2024-10-09T20:00:00Z",
-    "fortunaTime": "9999-12-01T05:00:00Z"
```

Comment:
why is Fortuna 9999?

Reply:
This is just some docs on a potential response... 9999 is the "unactivated" timestamp that we use.

Reply:
shouldn't we set it to 8th of April?